pkg/scheduler/objects/utilities_test.go

```diff
@@ -224,14 +224,14 @@ func newAllocationAskAll(allocKey, appID, taskGroup string, res *resources.Resou
 	return NewAllocationAskFromSI(ask)
 }
 
-func getTestUserGroup() security.UserGroup {
-	return security.UserGroup{User: "testuser", Groups: []string{"testgroup"}}
+func getTestUserGroup(userName string, groupNameList []string) security.UserGroup {
```

Comment:
Same here: use a separate function `getUserGroup(userName string, groupNameList []string) security.UserGroup` You can even make `getTestUserGroup()` call that one with the default test user details: `"testuser", []string{"testgroup"}`

Reply:
Addressed in latest PR.